packages/core/state/interaction/logics.ts

```diff
@@ -582,17 +582,81 @@ const setIsSmallScreen = createLogic({
  * Logs details of files that are being moved.
  */
 const moveFilesLogic = createLogic({
-    async process({ action, getState }: ReduxLogicDeps, _dispatch, done) {
+    async process({ action, getState }: ReduxLogicDeps, dispatch, done) {
         try {
             const httpFileService = interactionSelectors.getHttpFileService(getState());
             const username = interactionSelectors.getUserName(getState());
-            const fileIds = (action as MoveFilesAction).payload.fileDetails.map((file) => file.id);
-            const cacheStatuses = await httpFileService.cacheFiles(fileIds, username);
 
-            // TODO: What to do with the status
-            console.log("Cache statuses:", cacheStatuses);
+            const fileDetails = (action as MoveFilesAction).payload.fileDetails;
+
+            // Map file IDs to file names for easy lookup
+            const fileIdToNameMap = Object.fromEntries(
+                fileDetails.map((file) => [file.id, file.name])
+            );
+
+            // Extract file IDs
+            const fileIds = fileDetails.map((file) => file.id);
```

Comment:
was unsure about the necessity of the single read `const` when `fileIdToNameMap` seems to duplicate the info, but also understand if the purpose is readability/separation of function

Reply:
The idea was to make decoding the file id back to file name for error reporting more streamlined. I dont really have an opinion either way .